cdc/log.cc

```diff
@@ -746,6 +746,28 @@ class transformer final {
             };
 
             int batch_no = 0;
+            if (!p.static_row().empty()) {
+                std::optional<clustering_key> pikey;
+                const cql3::untyped_result_set_row * pirow = nullptr;
+
+                if (rs && !rs->empty()) {
+                    // For static rows, only one row from the result set is needed
+                    pikey = set_pk_columns(m.key(), ts, tuuid, batch_no, res);
+                    set_operation(*pikey, ts, operation::pre_image, res);
+                    pirow = &rs->front();
+                    ++batch_no;
+                }
+
+                auto log_ck = set_pk_columns(m.key(), ts, tuuid, batch_no, res);
+                auto ttl = process_cells(p.static_row().get(), column_kind::static_column, log_ck, pikey, pirow);
+
+                set_operation(log_ck, ts, operation::update, res);
+
+                if (ttl) {
+                    set_ttl(log_ck, ts, *ttl, res);
+                }
+                ++batch_no;
+            } else {
             for (const rows_entry& r : p.clustered_rows()) {
```

Comment:
Please add one more commit after this to fix the indent.

Reply:
The next commit (`cdc: fix indentation in transformer::transform`) already does exactly that.

Reply:
Ok. I missed that commit. Sorry.